src/Components/InputElement/InputElement.module.css

```diff
@@ -0,0 +1,21 @@
+.story {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
```

Comment:
I think you're using this only for your storybook - so it should not appear in the code for your app. 
You could use inline css here if you're only using it once or you could create a separate CSS file for your storybook.